NthKey/SupportingFiles/NthKey.xcdatamodeld/NthKey.xcdatamodel/contents

```diff
@@ -1,4 +1,27 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="15702" systemVersion="19C57" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
-    <elements/>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="17709" systemVersion="20E5196f" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="AddressEntity" representedClassName="AddressEntity" syncable="YES" codeGenerationType="class">
+        <attribute name="address" attributeType="String"/>
+        <attribute name="receiveIndex" attributeType="Integer 16" defaultValueString="0" usesScalarValueType="YES"/>
+        <attribute name="used" attributeType="Boolean" defaultValueString="NO" usesScalarValueType="YES"/>
+        <relationship name="wallet" maxCount="1" deletionRule="Nullify" destinationEntity="WalletEntity" inverseName="addresses" inverseEntity="WalletEntity"/>
+    </entity>
+    <entity name="CosignerEntity" representedClassName="CosignerEntity" syncable="YES" codeGenerationType="class">
+        <attribute name="derivation" optional="YES" attributeType="String"/>
+        <attribute name="fingerprint" optional="YES" attributeType="Binary"/>
+        <attribute name="name" optional="YES" attributeType="String"/>
+        <attribute name="xpub" optional="YES" attributeType="String"/>
+        <relationship name="wallet" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="WalletEntity" inverseName="cosigners" inverseEntity="WalletEntity"/>
+    </entity>
+    <entity name="WalletEntity" representedClassName="WalletEntity" syncable="YES" codeGenerationType="class">
+        <attribute name="network" optional="YES" attributeType="Integer 16" defaultValueString="0" usesScalarValueType="YES"/>
+        <attribute name="threshold" optional="YES" attributeType="Integer 16" defaultValueString="0" usesScalarValueType="YES"/>
+        <relationship name="addresses" optional="YES" toMany="YES" deletionRule="Nullify" ordered="YES" destinationEntity="AddressEntity" inverseName="wallet" inverseEntity="AddressEntity"/>
+        <relationship name="cosigners" toMany="YES" deletionRule="Nullify" destinationEntity="CosignerEntity" inverseName="wallet" inverseEntity="CosignerEntity"/>
```

Comment:
Better to delete cosigners when deleting the wallet. Unlike Specter, we don't support creating a wallet manually, so we always have to reimport this stuff when creating a fresh wallet.

This also means CosignerEntity's can be duplicates (other than their wallet reference). But `WalletEntity`s are unique.